calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -79,18 +83,75 @@ modify the `default` FelixConfiguration to set the field `policySyncPathPrefix`
 kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySyncPathPrefix":"/var/run/nodeagent"}}'
 ```
 
+#### (Optional) Deploy a sample application
+If you don’t have an application to test WAF with or don’t want to use it right away against your own application,
+we recommend that you install the [GoogleCloudPlatform/microservices-demo app](https://github.com/GoogleCloudPlatform/microservices-demo):
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/release/v0.7.0/release/kubernetes-manifests.yaml
+```
 ## How to
 
-- [Enable WAF](#enable-waf)
+- [Enable WAF on your cluster](#enable-waf-on-your-cluster)
+- [Apply WAF to your services](#apply-waf)
 - [View WAF events](#view-waf-events)
 - [Create global alerts](#create-global-alerts)
-- [Manage WAF](#manage-waf)
-   - [Change rules](#change-waf-rules)
-   - [Disable WAF for a service](#disable-waf-for-a-service)
 
-### Enable WAF
+### Enable WAF on your cluster
+
+Enabling WAF is a one-time activity. You can enable it either [through UI](#enable-waf-using-the-ui) or [CLIs](#enable-waf-using-kubectl-clis).
+
+#### Enable WAF using the UI
+
+On the Manager UI, click **Threat Defense**, **Web Application Firewall**, **Configure Web Application Firewall**.
+
+ <img
+    src='/img/calico-enterprise/waf-config-first-step.png'
+    alt='WAF services'
+    width='600'
+  />
+
+:::note
+
+If WAF is already enabled in your cluster, you do not see this page.
+:::
+
+#### Enable WAF using kubectl CLIs
+
+In the ApplicationLayer custom resource, named `tigera-secure`, set the `webApplicationFirewall` field to `Enabled`.
+
+```bash
```

Comment:
```suggestion
```yaml
```

Reply:
The entire block can be used as a `bash` command. So, I used `bash` there.